src/aura/BGE_DataImportBatchEntry/BGE_DataImportBatchEntry.cmp

```diff
@@ -0,0 +1,165 @@
+    <!--Events-->
+    <aura:handler name="init" action="{!c.doInit}" value="{!this}"/>
+    <aura:handler event="ltng:sendMessage" action="{!c.handleMessage}"/>
+
+    <!--Lightning Data Service for Batch Record-->
+    <aura:if isTrue="{!v.isLoaded}">
+        <force:recordData aura:id="forceRecordCmp"
+                          recordId="{!v.recordId}"
+                          fields="{!v.batchFields}"
+                          targetFields="{!v.record}"
+        />
+    </aura:if>
+
+    <!--Start Markup-->
+
+    <lightning:notificationsLibrary aura:id="notifLib"/>
+
+    <lightning:layout multipleRows="true">
+
+        <!--Start New Row-->
+        <lightning:layoutItem size="12" class="slds-p-bottom_x-small">
+            <lightning:card title="{!$Label.c.bgeGridNewGift}" aura:id="entryFormCard" class="bgeCard">
+                <lightning:spinner aura:id="formSpinner" alternativeText="{!$Label.c.labelMessageLoading}"/>
+                <!--BGE_EntryForm dynamically rendered here-->
+                {!v.body}
+            </lightning:card>
+        </lightning:layoutItem>
+        <!--End New Row-->
+
+        <!--Start BGE Table-->
+        <lightning:layoutItem size="12" class="slds-p-vertical_x-small">
+            <lightning:card title="{!$Label.c.bgeGridAllGifts}" class="bgeCard">
+
+                <lightning:spinner aura:id="dataTableSpinner" alternativeText="{!$Label.c.labelMessageLoading}"/>
+
+                <!--Expected totals-->
+                <lightning:layout multipleRows="true">
+
+                    <lightning:layoutItem size="12" padding="horizontal-small">
+                        <!--<aura:if isTrue="{!greaterthan(mult(100,div(v.totals.countGifts,v.record.Expected_Count_of_Gifts__c)),100)}">
+                            Warning: Total gift count exceeds expected gift count. <br/> </aura:if>-->
+
+                        <div class="slds-grid slds-grid_align-spread slds-p-bottom_x-small">
+                            <span>{!$Label.c.bgeGridGiftCountHeader}</span>
+                            <span>
+                                <strong>
+                                    <aura:if isTrue="{! !v.isNamespaced}">
+                                        {!format($Label.c.bgeGridTotalCount, v.totals.countGifts, v.record.Expected_Count_of_Gifts__c)}
+                                    </aura:if>
+                                    <aura:if isTrue="{!v.isNamespaced}">
+                                        {!format($Label.c.bgeGridTotalCount, v.totals.countGifts, v.record.npsp__Expected_Count_of_Gifts__c)}
+                                    </aura:if>
+                                </strong>
+                            </span>
+                        </div>
+                        <!--Progress bar is on a 100 point scale, so just using division doesn't work here-->
+                        <aura:if isTrue="{! !v.isNamespaced}">
```

Comment:
I prefer `{!not(v.isNamespaced)}` to avoid confusion with the double meaning of !.